### ISSUE TITLE
run analyses for two additional 30-year time periods

### ISSUE THREAD

**Issue by teixeirak** [SCBI-ForestGEO/climate_sensitivity_cores#60]
Let's run analyses for three consecutive 30-year windows (1910-1939, 1940-1979, and 1980-2009). Species should be dropped when they have SSS<0.75 for any part of the period (which means we'll miss a lot in 1910-1939). 

We will probably end up making a new figure with these.

**Comment by ValentineHerr** [comment]
I added the new periods, let me know what you think and what to do next with these.
Note, I had to adjust the upper range of the correlations in the quilt plots, to accommodate higher values in 1940-1979.

**Comment by teixeirak** [comment]
WOW!  That's dramatic. Change in PET sensitivity:
early- 
![image](https://user-images.githubusercontent.com/6355854/55251354-c2167f80-5226-11e9-86ab-b247a58f6b35.png)
mid- 
![image](https://user-images.githubusercontent.com/6355854/55251403-df4b4e00-5226-11e9-89b2-425318115208.png)

late- 
![image](https://user-images.githubusercontent.com/6355854/55251380-d35f8c00-5226-11e9-834e-e282b256a129.png)

**Comment by ValentineHerr** [comment]
Note that the figures for 1980_2009 and 1901_2016 also have adjusted color scales to go up to 1.2

**Comment by teixeirak** [comment]
This is a really interesting result-- one that we'll definitely want to show. I want to plan this out well before we start formatting into figures/ tables for publication. 
To get a better idea of the climate in these three periods, could you please modify the climate figure [here](https://github.com/SCBI-ForestGEO/climate_sensitivity_cores/tree/master/results/climate) to show the 4 different time periods? Hopefully that won't be too busy. Perhaps use red,blue,yellow for the 3 time periods + bold black for the mean.

**Comment by teixeirak** [comment]
Oops! I just realized that we did the math wrong here. 1940-1979 is 40 years. So sorry about that.

Instead, let's do 3 33-yr periods:
1977-2009
1944-1976
1911-1943

Conveniently, each of these includes 3-4 of the 10 strongest droughts identified [here](https://github.com/SCBI-ForestGEO/climate_sensitivity_cores/issues/59):

10 largest PET-PRE: 82.63, 81.9, 112.43, 84.57, 89.43, 83.87, 83.37, 86.97, 80,
82.13
Corresponding drought years: 1911, 1914, 1930, 1936, 1944, 1964, 1966, 1977, 1999, 2007
